wemake_python_styleguide/violations/annotations.py

```diff
@@ -91,3 +93,34 @@ class NestedAnnotationsViolation(ASTViolation):
 
     error_template = 'Found redundant nested typing annotation'
     code = 702
+
+
+@final
+class UnionNoneViolation(ASTViolation):
+    """
+    Forbids to use ``Union[T, None]`` typing annotation.
+
+    Reasoning:
+        Union[T, None] is just the same as Optional[T].
+        There's no need to use the first version.
+        It is not type related, it is a consistency rule.
+
+    Solution:
+        Replace ``Union[T, None]`` with ``Optional[T]``.
+
+    Example::
+
+        # Correct:
+        def func(empty: Optional[T]):
```

Comment:
Let's use variables as examples. They take less space. And no problem with `'''`